### ISSUE TITLE
Suggestion for addition of note in Forms section of docs

### ISSUE THREAD

**Issue by joncarr** [gobuffalo/gobuffalo#349]
If somebody elects to not use the form helper then simply putting "PUT" as a method type for their Update action will not successfully update the database despite giving no errors.  Behind the scenes in `form.go` Buffalo is checking this and if the method is not GET or POST, a hidden field is being added to overwrite the form action.  As can be seen [HERE](https://github.com/gobuffalo/tags/blob/da5de2a6b4fdd40153789f7efff1330616e28de7/form/form.go#L44).

Suggestion: Add a note in the documentation that lets the user know that if they elect to not use the form helper (for whatever reason), then they need to structure their form as such for an UPDATE action:
```
<form method="post" ...>
  <input type="hidden" name="_method" value="put" />
...
```

**Comment by lukasschlueter** [comment]
The form helpers also take care of the authenticity_token. See [this](https://github.com/gobuffalo/tags/blob/da5de2a6b4fdd40153789f7efff1330616e28de7/form/form.go#L15) method.

I'm not sure if and how it is possible to add this token manually. If this is not possible, I think it would be good to also include a hint for that.

Can someone confirm if forms without form helper work despite the csrf tokens?

**Comment by joncarr** [comment]
You can add a helper in `render.go` for your csrf token.  The code for that would look something like:
```
"csrf": func() template.HTML {
		return template.HTML("<input name=\"authenticity_token\" value=\"<%= authenticity_token %>\" type=\"hidden\">")
},
```

Inside you would use the csrf helper in your template with `<%= csrf() %>`

**Comment by lukasschlueter** [comment]
Thanks :+1:

In what situation could I want to circumvent the form helper? Is that really worth it or would defining a custom form helper make more sense? 

I'm struggling with coming up with a good formulation for the 'how?' without being able to include a 'why?'

Did you use this yourself?

**Comment by joncarr** [comment]
Unable to reference currently...
[PR #372](https://github.com/gobuffalo/gobuffalo/pull/372)

**Comment by stanislas-m** [comment]
Fixed by #372.
